Solutions/net40/dllmain.cpp

```diff
@@ -24,7 +24,7 @@ void LockInMemory()
 {
 #ifndef _WINRT_DLL
     HMODULE mstelModule;
-    ::GetModuleHandleExW(GET_MODULE_HANDLE_EX_FLAG_PIN, L"ClientTelemetry.dll", &mstelModule);
+    ::GetModuleHandleExW(GET_MODULE_HANDLE_EX_FLAG_PIN, L"ClientTelemetry3m.dll", &mstelModule);
```

Comment:
Actually maybe we need to revert this. The name of the DLL would be different depending on SKU..